components/common/IconSwitch.tsx

```diff
@@ -54,7 +56,7 @@ export const IconSwitch: FC<IconSwitchProps> = ({
               : theme.iconDisabled,
           )}
         >
-          {icon}
+          {icon || iconActive}
```

Comment:
`onIcon` should be displayed when `enabled` is `true`, otherwise `offIcon` should be displayed